package.json

```diff
@@ -1,89 +1,137 @@
-    "lint": "lerna run lint",
-    "lint-fix": "lerna run lint -- --fix",
-    "postclean-all": "yarn sf-clean all",
-    "postinstall": "yarn bootstrap",
-    "prepack": "lerna run prepack",
-    "test": "lerna run test",
-    "test:nuts": "lerna run test:nuts"
+    "build": "sf-build",
+    "clean": "sf-clean",
+    "clean-all": "sf-clean all",
+    "clean:lib": "shx rm -rf lib && shx rm -rf coverage && shx rm -rf .nyc_output && shx rm -f oclif.manifest.json",
+    "compile": "sf-compile",
+    "docs": "sf-docs",
+    "format": "sf-format",
+    "lint": "sf-lint",
+    "postpack": "shx rm -f oclif.manifest.json",
+    "posttest": "yarn lint && yarn test:deprecation-policy && yarn test:command-reference",
+    "prepack": "sf-prepack",
+    "prepare": "sf-install",
+    "pretest": "sf-compile-test",
+    "test": "sf-test",
+    "test:command-reference": "./bin/run commandreference:generate --erroronwarnings",
+    "test:deprecation-policy": "./bin/run snapshot:compare",
+    "test:nuts": "nyc mocha \"**/*.nut.ts\" --slow 4500 --timeout 600000 --parallel",
+    "version": "oclif-dev readme"
+  },
+  "dependencies": {
+    "@oclif/config": "^1",
+    "@salesforce/command": "^3.1.3",
+    "@salesforce/core": "^2.24.0",
+    "@salesforce/data": "^0.1.8",
+    "@salesforce/ts-types": "^1.5.13",
+    "@types/graceful-fs": "^4.1.5",
+    "@types/mkdirp": "^1.0.1",
+    "chalk": "^4.1.0",
+    "csv-parse": "^4.14.1",
+    "tslib": "^2"
   },
   "devDependencies": {
     "@oclif/dev-cli": "^1",
+    "@oclif/plugin-command-snapshot": "^2.1.1",
     "@salesforce/cli-plugins-testkit": "^1.1.5",
     "@salesforce/dev-config": "^2.1.2",
-    "@salesforce/dev-scripts": "0.9.15",
     "@salesforce/plugin-command-reference": "^1.3.0",
+    "@salesforce/dev-scripts": "0.9.15",
     "@salesforce/prettier-config": "^0.0.2",
-    "@salesforce/ts-sinon": "^1.3.12",
-    "@typescript-eslint/eslint-plugin": "4.26.0",
-    "@typescript-eslint/parser": "4.26.0",
+    "@salesforce/ts-sinon": "^1.3.15",
+    "@types/chai-as-promised": "^7.1.3",
+    "@types/jsforce": "^1.9.29",
+    "@types/sinon": "^10.0.2",
+    "@typescript-eslint/eslint-plugin": "^4.26.0",
+    "@typescript-eslint/parser": "^4.26.0",
     "chai": "^4.2.0",
+    "chai-as-promised": "^7.1.1",
+    "cz-conventional-changelog": "^3.3.0",
     "eslint": "^7.27.0",
-    "eslint-config-prettier": "^8.3.0",
+    "eslint-config-prettier": "^6.11.0",
     "eslint-config-salesforce": "^0.1.6",
     "eslint-config-salesforce-license": "^0.1.6",
     "eslint-config-salesforce-typescript": "^0.2.7",
-    "eslint-plugin-header": "^3.0.0",
-    "eslint-plugin-import": "^2.20.2",
-    "eslint-plugin-jsdoc": "^35.0.0",
+    "eslint-plugin-header": "^3.1.1",
+    "eslint-plugin-import": "^2.23.4",
+    "eslint-plugin-jsdoc": "^35.1.2",
     "eslint-plugin-prettier": "^3.1.3",
-    "husky": "^4.3.8",
-    "lerna": "^4.0.0",
-    "mocha": "^8.3.0",
+    "fast-xml-parser": "^3.17.5",
+    "husky": "^5.1.3",
```

Comment:
leave the husky on v4 (major v5 has breaking changes, and we'll have to update dev-scripts and all the repos at one time to cross a major version)